test/viewmodels/helpers/messages/ViewMessageHelperSpec.scala

```diff
@@ -800,6 +802,78 @@ class ViewMessageHelperSpec extends SpecBase
 
     }
 
+    "for an IE819" must {
+
+      "return the correct content when the errors are non-fixable, 3rd party submission" in {
+        val failureMessageResponse = GetSubmissionFailureMessageResponse(
+          ie704 = IE704ModelFixtures.ie704ModelModel.copy(
+            body = IE704BodyFixtures.ie704BodyModel.copy(functionalError = Seq(
+              IE704FunctionalError(
+                errorType = "4403",
```

Comment:
Should this errorType match an IE819 error code?

Reply:
It doesn't really matter because we are testing the content below the error summary list. However, if you think it's worth it, I can add the applicable error code?